development/README.md

```diff
@@ -1,127 +1,22 @@
-
-The account files are encrypted with the **ENCRYPTION_KEY_NAME** key from **KEYRING_NAME** and are stored in **BUCKET_NAME**.
-
-3. Run the following script to start the installation process:
-
-```bash
-./install-prow.sh
-```
-
-The installation script performs the following steps to install Prow:
-
-- Deploy the NGINX Ingress Controller.
-- Create a ClusterRoleBinding.
-- Create a HMAC token to be used for GitHub Webhooks.
-- Create secrets for HMAC and OAuth2 to be used by Prow.
-- Deploy Prow components using the `starter.yaml` file from the `prow/cluster` directory.
-- Add annotations for the Prow Ingress to make it work with the NGINX Ingress Controller.
-
-To check if the installation is successful, perform the following steps:
-
-1. Check if all Pods are up and running:
-   `kubeclt get pods`
-2. Check if the Deck is accessible from outside of the cluster:
-   `kubectl get ingress ing`
-   Copy the address of the ingress `ing` and open it in a browser to display the Prow status on the dashboard.
-
-## Development
-
-[Configure Webhook](https://github.com/kubernetes/test-infra/blob/master/prow/getting_started.md#add-the-webhook-to-github) to enable sending Events from a GitHub repository to Prow.
-When you use our script `install-prow.sh` to install Prow on your cluster the list of plugins and configuration is empty. You can configure Prow by specifying the `plugins.yaml` and `config.yaml` files.
-
-To check if the `plugins.yaml` and `config.yaml` configuration files are correct, run the `check.sh` script.
-In case of changes in the plugins configuration, use the `update-plugins.sh` to apply changes on a cluster.
-In case of changes in the jobs configuration, use the `update-config.sh` to apply changes on a cluster.
-
-### Strategy for organising jobs
-
-The `test-infra` repository has defined configurations for the Prow cluster in the `prow` subdirectory. This directory has the following structure:
-
-- `cluster` directory, which contains all `yaml` files for Prow cluster provisioning
-- `jobs/{repository_name}` directory, which contains all files with jobs definitions. Each file must have a unique name.
-- `config.yaml` file, which contains Prow configuration without job definitions
-- `plugins.yaml` file, which contains Prow plugins configuration
-
-`jobs/{repository_name}` directories have subdirectories which represent each component and contain job definitions. Job definitions not connected to a particular component, like integration jobs, are defined directly under the `jobs/{repository_name}` directory.
-
-For example:
-
-```
-...
-prow
-|- cluster
-| |- starter.yaml
-|- jobs
-| |- kyma
-| | |- components
-| | | |- environments
-| | | | |- environments.jobs.yaml
-| | |- kyma.integration.yaml
-|- config.yaml
-|- plugins.yaml
-...
 ```
+  ├── checker                  # Code sources of a simple Go application that verifies the configuration of the "config.yaml" and "plugins.yaml" files.          
```

Comment:
@michal-hudy all that files refers to prow, shouldn't we move development directory to prow/development?

Reply:
Good point.

Reply:
I have created a follow-up issue for that: #117